dataprofiler/labelers/data_processing.py

```diff
@@ -825,8 +825,8 @@ def _word_level_argmax(self, data, predictions, label_mapping,
             label_count = {label_mapping[default_label]: 0}
             for idx in range(len(sample)):
 
-                # Split on separator
-                if sample[idx] in separator_dict:
+                # Split on separator or last sample
+                if sample[idx] in separator_dict or idx == len(sample)-1:
```

Comment:
should we add a test for this?

Reply:
Very good point, I'll add a test to evaluate this case